lbz/events/broker.py

```diff
@@ -2,42 +2,35 @@
 from typing import Callable, Dict, List
 
 from lbz.events.event import Event
+from lbz.handler_base import HandlerBase
 from lbz.misc import get_logger
 
 logger = get_logger(__name__)
 
 
-class EventBroker:
+class EventBroker(HandlerBase):
     def __init__(
         self,
         mapper: Dict[str, List[Callable]],
-        raw_event: dict,
+        event: dict,
         *,
+        context: object = None,
```

Comment:
Context can be provided as a positional argument:
```suggestion
        event: dict,
        context: object = None,
        *,
```